package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "uk.co.compucorp.civicase",
+  "version": "2.0.0",
+  "description": "Civicase Extension based on Shoreditch theme",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs",
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
+      "semistandard --fix",
+      "git add"
+    ],
+    "*.scss": [
+      "stylelint --fix",
+      "git add"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/compucorp/uk.co.compucorp.civicase.git"
+  },
+  "author": "compucorp.co.uk",
```

Comment:
We should provide the company name `"Compucorp <info@compucorp.co.uk>"`

Reply:
Cool. Updated.